src/components/buttons/HdButton.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <button :class="computedClasses">
```

Comment:
Did you consider exposing also some `button` attributes(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Attributes)? I would suggest adding at least `type` - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type as prop. 

If you want to avoid setting each one, it could also take the attributes object that then dynamically renders passed in attributes.

Reply:
Don't think this is needed since the `button` is a root component, so any attributes added to the component in the template will be "rendered" on it.

Reply:
We won't need to add `<button>` attributes for the reason @OriginalEXE pointed. If we pass `:disabled="disabled"` or so it will be added to the root HTML tag which is the `<button>` itself.

Reply:
Ah true, it's root. All good. :+1: